src/python/grpcio/grpc/_server.py

```diff
@@ -390,6 +398,16 @@ def unary_request():
     return unary_request
 
 
+def _get_default_initial_metadata_operation(state):
```

Comment:
nit: Should we follow the define-before-use pattern, and move this function above `_abort`?

Reply:
Fair point. Counterpoint: This file is so large that "comes-before" and "comes-after" relationships have no meaning. It's not practical to scroll through. Every time I've navigated this file, I've done so by jumping between function definitions. Less of an argument against your suggestion and more of an argument against such large files. Fixed.